maryam

```diff
@@ -41,37 +32,45 @@ def cui(args, format_help):
 		if option in ['-v', '--version']:
 			print(f"OWASP MARYAM V.{__VERSION__}")
 			exit(0)
-		command = ' '.join(args[1:])
-		if not v or option not in ['-e', '--execute', '-s', '--start']:
+		if not option or option not in ['-e', '--execute', '-s', '--start']:
```

Comment:
I think checking for `not option` is redundant since inside this `if` there will always be arguments